### ISSUE TITLE
feature request (enhancement) // support for windowed fullscreen

### ISSUE THREAD

**Issue by Mistreaper** [AsherGlick/Burrito#59]
## Why?
When using windowed fullscreen in Guild Wars 2, alt tabbing to the game makes burrito invisible. 

I tried using `wmctrl` (you may replace `:SELECT:` with a substring match) instead of "keep above others":
```
wmctrl -r :SELECT: -b add,above
```
But the issue still persists (I don't know why). I also found another issue (#53) where the overlay also overlays over other windows. It's very annoying playing a game in window mode instead of windowed fullscreen. 

I think this is the "most needed" feature other than easier setup of burrito links, etc, etc.

**Comment by AsherGlick** [comment]
If you know of a better way to handle the overlay capabilities, I would be happy to review and merge any upgrades you would like to submit. You can find the current primary x11 overlay logic is stored within the `burrito-fg` module. 

Specifically x11 interactions is found in the function
https://github.com/AsherGlick/Burrito/blob/60927eff6a91d62175fe0f8a4c823654f384065b/burrito-fg/src/lib.rs#L32

which is called from the line
https://github.com/AsherGlick/Burrito/blob/60927eff6a91d62175fe0f8a4c823654f384065b/Spatial.gd#L290

Windowed fullscreen works without issue in my normal usage, so this may be a DE specific problem. I have hardware setup to test in Ubuntu 20.04 running gnome, all other DEs are community supported at this time.

**Comment by AsherGlick** [comment]
@Melechtna Have you had success in running GW2 inside a VM like qemu or virtualbox without passing through a GPU? If so that might be a viable testing method for me to experiment with.

**Comment by Melechtna** [comment]
> @Melechtna Have you had success in running GW2 inside a VM like qemu or virtualbox without passing through a GPU? If so that might be a viable testing method for me to experiment with.

Yes I have, though I specifically use KVM through Virt-Manager, the performance is FAR from great, but it runs well ENOUGH using QXL for tests like these.

**Comment by Melechtna** [comment]
> @Melechtna Have you had success in running GW2 inside a VM like qemu or virtualbox without passing through a GPU? If so that might be a viable testing method for me to experiment with.

ALSO, you could just run an LXC through Virt-Manager, which is basically just Linux in Linux, which should hypothetically have no bottle necks beyond the ones you set yourself when configuring it and the resources it has access too.

**Comment by Technetium1** [comment]
@Melechtna that still hits the same wall for [passing through hardware](https://www.reddit.com/r/Proxmox/comments/glog5j/lxc_gpu_passthrough/).
